rcl/test/rcl/test_events.cpp

```diff
@@ -730,6 +730,46 @@ TEST_F(TestEventFixture, test_bad_get_handle)
   EXPECT_EQ(NULL, rcl_event_get_rmw_handle(NULL));
 }
 
+/*
+ * Basic test subscriber event message lost
+ */
+TEST_F(TestEventFixture, test_sub_message_lost_event)
+{
+  const rmw_qos_profile_t subscription_qos_profile = default_qos_profile;
+
+  rcl_ret_t ret = setup_subscriber(subscription_qos_profile);
+  ASSERT_EQ(ret, RCL_RET_OK) << rcl_get_error_string().str;
+
+  // Check if supported
+  subscription_event = rcl_get_zero_initialized_event();
+  ret = rcl_subscription_event_init(
+    &subscription_event,
+    &subscription,
+    RCL_SUBSCRIPTION_MESSAGE_LOST);
+  EXPECT_TRUE(ret == RCL_RET_OK || ret == RCL_RET_UNSUPPORTED);
```

Comment:
instead of an `||`, please check `RCL_RET_UNSUPPORTED` when `rmw_fastrtps_*` is running (which is the implementation that doesn't support the feature).

Reply:
Changed with 925fb89